sdk/cosmosdb/cosmos/src/utils/types.d.ts

```diff
@@ -0,0 +1 @@
+export type VerboseOmit<T, K extends keyof any> = Pick<T, Exclude<keyof T, K>>;
```

Comment:
A comment along the lines of "Shim for Omit which was added in TypeScript 3.5" would be helpful for the future1